Applications/Training/README.md

```diff
@@ -2,20 +2,85 @@
 
 Unlike using kNN for the classifier at #55 , we could add new layer at the end of feature extractor.
 In this toy example, fully connected layer is added to classify as below. The size of the feature is also same with previous example and the training set and test set are also same. ( 3 classes, 5 training set and 2 test set of each )
-Only fully connected layer is updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
-I wrote some code for the forward and backward propagation during fully connected layer training.
+Only fully connected layers are updatable and Mobilenet ssd v2 is used for feature extractor like previous example and all the testing and training is done on the Galaxy S8.
 
-Sigmoid function is used for the activation. Square error loss function and gradient decent is used for the loss function and optimizer. Just 20 neuron is used for the hidden layer and iterate 300 times to make is more simple.
+Sigmoid function is used for the activation. Square error loss function and gradient decent is used for the loss function and optimizer. Two fully connected layers are added for the hidden layer and iterate 300 times to make is more simple.
```

Comment:
`Sigmoid function is used for the activation.` -> lets improve this. Its quite vague without knowing which layer we are talking about. I thought it was the final activation but that's softmax.
either way, if we talk about activation, it must say where that activation is.

Reply:
fixed. thanks.